mk/generate.mk

```diff
@@ -46,12 +46,18 @@ resources/type: $(RESOURCE_GEN)
 clean/resources:
 	find pkg -name 'zz_generated.*.go' -delete
 
-POLICIES_DIR := pkg/plugins/policies
+POLICIES_DIR ?= pkg/plugins/policies
 COMMON_DIR := api/common
 
-policies = $(foreach dir,$(shell find pkg/plugins/policies -maxdepth 1 -mindepth 1 -type d | grep -v -e core | sort),$(notdir $(dir)))
+policies = $(foreach dir,$(shell find $(POLICIES_DIR) -maxdepth 1 -mindepth 1 -type d | grep -v -e '/core$$' | grep -v -e '/system$$' | grep -v -e '/mesh$$' | sort),$(notdir $(dir)))
 kuma_policies = $(foreach dir,$(shell find $(KUMA_DIR)/pkg/plugins/policies -maxdepth 1 -mindepth 1 -type d | grep -v -e core | sort),$(notdir $(dir)))
 
+generate/resources: POLICIES_DIR=pkg/core/resources/apis
+generate/resources:
+	POLICIES_DIR=$(POLICIES_DIR) $(MAKE) $(addprefix generate/policy/,$(policies))
+	POLICIES_DIR=$(POLICIES_DIR) $(MAKE) generate/policy-import
+	POLICIES_DIR=$(POLICIES_DIR) HELM_VALUES_FILE_POLICY_PATH=".plugins.resources" $(MAKE) generate/policy-helm
```

Comment:
All this very alike. Shouldn't we have `RESOURCES_DIR` and have both pkg/core/resources/apis and pkg/plugins/policies?

Reply:
make sense, let me try this

Reply:
Let's do this in a separate PR @lahabana ?